.github/workflows/dis-syncroot-release.yml

```
@@ -0,0 +1,64 @@
+            if command -v kubectl >/dev/null 2>&1; then
+              kubectl kustomize "$path" >/dev/null
+              return
+            fi
+            echo "Error: Neither 'kustomize' nor 'kubectl' found in runner PATH."
+            exit 1
+          }
+
+          build_kustomize "./syncroot/${SYNCROOT_ENVIRONMENT}"
+
+      - name: Build and push syncroot artifact
+        uses: ./actions/flux/build-push-image
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/actions/flux/build-push-image": line 5: unexpected key "type" for definition of input "azure_tenant_id"

(action)